Makefile

```diff
@@ -97,7 +97,7 @@ build: manifests generate fmt vet lint ## Build manager binary.
 	go build -o $(OUT_DIR)/package-operator ./cmd/package-operator/
 
 .PHONY: build-cli
-build-cli: fmt vet lint ## Build cli binary.
+build-cli: fmt vet lint lint-web web ## Build cli binary.
```

Comment:
I think the `lint` target should now be called `lint-go`, or if it stays `lint`, it should also execute `lint-web`. But in the latter the `lint-fix` would also need to include a new target `lint-fix-web` (which itself would probably execute `prettier:write`?).